demo/bin/pma-demo-update

```diff
@@ -69,19 +69,20 @@ for REPONAME in $REPOS ; do
         composer update --quiet --no-dev
     fi
 
-    if [ -f package.json ] ; then
-        # We still redirect output to /dev/null because even with the --silent flag, yarn still gives sass output such as
-        # $ sass themes/pmahomme/scss:themes/pmahomme/css themes/original/scss:themes/original/css themes/metro/scss:themes/metro/css
-        yarn install --non-interactive --silent > /dev/null
-    fi
-
     # Link themes
     for THEME in $THEMES ; do
         if [ ! -e themes/$THEME ] ; then
             ln -s ../../themes/$THEME themes/$THEME
         fi
     done
 
+    if [ -f package.json ] ; then
+        yarn install --non-interactive --silent --ignore-scripts
+        if [ ${REPONAME%%-*} = 'master' ] ; then
```

Comment:
Maybe there is better way to detect need to run this to avoid need to change the code once this code gets into maintenance branches?

Reply:
Yeah, I know. That's why I didn't merge this PR. I resolved this by adding `--silent` to the package.json script.

https://github.com/phpmyadmin/phpmyadmin/blob/205c654274cdf9fb26058159da97f51299fddee9/package.json#L31